### ISSUE TITLE
Loss scale optimizer does not increment iterations

### ISSUE THREAD

**Issue by itmo153277** [keras-team/keras#20878]
Loss scale optimizer does not automatically increment iteration counter when `apply` method is called.

As a side effect, this breaks `evaluation_loss_vs_iterations` metric in TensorBoard when mixed precision is enabled.

See example here: [colab](https://colab.research.google.com/gist/itmo153277/295f75633b6b4f5dd9485027a93ec5a6/lossscaleiterations.ipynb)

**Comment by jeffcarp** [comment]
Thanks for the report, confirmed this fixes your original colab. As a workaround before this is released you can monkey patch the fix:

```
@property
def new_iterations(self):
  return self.inner_optimizer.iterations

keras.optimizers.LossScaleOptimizer.iterations = new_iterations
```
